OpenRA.Mods.Cnc/AudioLoaders/AudLoader.cs

```diff
@@ -25,7 +25,7 @@ bool IsAud(Stream s)
 			var readFormat = s.ReadByte();
 			s.Position = start;
 
-			if (!Enum.IsDefined(typeof(SoundFlags), readFlag))
+			if (readFlag > (int)SoundFlags._16Bit)
```

Comment:
These checks should match the actual file loading, so this should be removed and a new WestwoodCompressed check added after the SoundFormat check below.

Reply:
That won't change anything, though. There is already a `try catch` which returns false.

Reply:
This is about code maintainability rather than code logic.